README.md

```diff
@@ -105,6 +105,10 @@ which can be useful for you. Other related repositories are:
 * [GrimoireLab Tutorial](https://github.com/chaoss/grimoirelab-tutorial)
 * [GrimoireLab as a whole](https://github.com/chaoss/grimoirelab) (this repository)
 
+# Want to contribute to GrimoireLab?
+
+These are some general guidelines and information related to how we contribute to GrimoireLab: See [Contributing to GrimoireLab](https://github.com/chaoss/grimoirelab/blob/master/CONTRIBUTING.md)
+
```

Comment:
Please, move this section to the end of the README file. Otherwise, the next subsection (`Contents of this repository`) would be part of this new main section you added.

Reply:
Thanks for your input! I've made the necessary changes.